cmd/gitops/main.go

```diff
@@ -95,14 +98,15 @@ func main() {
 	fluxClient.SetupBin()
 	rootCmd.PersistentFlags().BoolVarP(&options.verbose, "verbose", "v", false, "Enable verbose output")
 	rootCmd.PersistentFlags().String("namespace", "wego-system", "gitops runtime namespace")
+	rootCmd.PersistentFlags().StringVarP(&options.endpoint, "endpoint", "e", os.Getenv("WEAVE_GITOPS_ENTERPRISE_API_URL"), "The Weave GitOps Enterprise HTTP API endpoint")
```

Comment:
Minor: I think people might find it a bit confusing to have `--endpoint` as an argument for every command. In fairness, the same could be said of `--namespace`, but that applies _almost_ everywhere.

Reply:
I agree that it can be confusing but they don't need to worry about it if they are not enterprise users. Maybe we can make that part more clear in the parameter description? The alternative would be to include that in every enterprise command.